lib/pka_mem.c

```diff
@@ -45,6 +50,27 @@ static pka_mem_desc_t *pka_data_mem_tbl[PKA_MAX_NUM_RINGS];
 //   33       176 - 183
 //   34       184
 
+static int pka_mem_lock()
+{
+    int result = 0;
+    if (NULL == p_mutex)
+    {
+	p_mutex = malloc(sizeof(pthread_mutex_t));

```

Comment:
Where is this deallocated ?

Reply:
this should be the life time of the process and shall be freed along when process exits.